packages/client/lib/CredentialRequestV1_0_09ClientBuilder.ts

```diff
@@ -1,28 +1,29 @@
 import {
   AccessTokenResponse,
-  CredentialOfferRequestPayloadV11,
+  CredentialOfferPayload,
   CredentialOfferRequestWithBaseUrl,
+  CredentialOfferV1_0_09,
   EndpointMetadata,
-  IssuanceInitiationRequestPayloadV9,
+  getIssuerFromCredentialOfferPayload,
   OpenID4VCIServerMetadata,
 } from '@sphereon/openid4vci-common';
 import { CredentialFormat } from '@sphereon/ssi-types';
 
+import { CredentialRequestClient } from './CredentialRequestClient';
 import { convertURIToJsonObject } from './functions';
-import {CredentialRequestClient} from "./CredentialRequestClient";
 
-export class CredentialRequestClientBuilder {
+export class CredentialRequestV1_0_09ClientBuilder {
   credentialEndpoint?: string;
   credentialType?: string | string[];
   format?: CredentialFormat | CredentialFormat[];
   token?: string;
 
-  public static fromURI({ uri, metadata }: { uri: string; metadata?: EndpointMetadata }): CredentialRequestClientBuilder {
-    return CredentialRequestClientBuilder.fromCredentialOfferRequest({
+  public static fromURI({ uri, metadata }: { uri: string; metadata?: EndpointMetadata }): CredentialRequestV1_0_09ClientBuilder {
+    return CredentialRequestV1_0_09ClientBuilder.fromCredentialOfferRequest({
```

Comment:
we can have a builder with two different methods. definitely. but it will make a messy builder. because for example while in the [v9](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0-09.html#name-issuance-initiation-request) we have `pre-authorized_code` and `user_pin_required`we have these in the grants in [v11](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0-11.html#name-credential-offer-parameters). the other properties are slightly different as well. so for me it makes more sense to isolate v9 codes here. because continuing the support for it in the main code will cause a lot of confusion later